malariagen_data/anoph/h12.py

```diff
@@ -514,6 +514,305 @@ def plot_h12_gwss(
+        else:
+            return fig
+
+    @check_types
+    @doc(
+        summary="Plot h12 GWSS data with multiple traces overlaid.",
+    )
+    def plot_h12_gwss_multi_overlay(
+        self,
+        contig: base_params.contig,
+        cohorts: base_params.cohorts,
+        window_size: h12_params.multi_window_size,
+        cohort_size: Optional[base_params.cohort_size] = h12_params.cohort_size_default,
+        sample_query: Optional[base_params.sample_query] = None,
+        analysis: hap_params.analysis = base_params.DEFAULT,
+        min_cohort_size: Optional[
+            base_params.min_cohort_size
+        ] = h12_params.min_cohort_size_default,
+        max_cohort_size: Optional[
+            base_params.max_cohort_size
+        ] = h12_params.max_cohort_size_default,
+        sample_query_options: Optional[base_params.sample_query_options] = None,
+        sample_sets: Optional[base_params.sample_sets] = None,
+        colors: gplt_params.colors = bokeh.palettes.d3["Category10"][10],
+        random_seed: base_params.random_seed = 42,
+        title: Optional[gplt_params.title] = None,
+        sizing_mode: gplt_params.sizing_mode = gplt_params.sizing_mode_default,
+        width: gplt_params.width = gplt_params.width_default,
+        track_height: gplt_params.track_height = 170,
+        genes_height: gplt_params.genes_height = gplt_params.genes_height_default,
+        show: gplt_params.show = True,
+        output_backend: gplt_params.output_backend = gplt_params.output_backend_default,
+    ) -> gplt_params.figure:
+        # Plot GWSS track.
+        fig1 = self.plot_h12_gwss_multi_overlay_track(
+            contig=contig,
+            sample_query=sample_query,
+            cohorts=cohorts,
+            cohort_size=cohort_size,
+            window_size=window_size,
+            analysis=analysis,
+            min_cohort_size=min_cohort_size,
+            max_cohort_size=max_cohort_size,
+            sample_query_options=sample_query_options,
+            sample_sets=sample_sets,
+            colors=colors,
+            random_seed=random_seed,
+            title=title,
+            sizing_mode=sizing_mode,
+            width=width,
+            height=track_height,
+            show=False,
+            output_backend=output_backend,
+        )
+
+        fig1.xaxis.visible = False
+        fig1.legend.location = "top_right"
+        fig1.legend.click_policy = "hide"
+
+        # Plot genes.
+        fig2 = self.plot_genes(
+            region=contig,
+            sizing_mode=sizing_mode,
+            width=width,
+            height=genes_height,
+            x_range=fig1.x_range,
+            show=False,
+            output_backend=output_backend,
+        )
+
+        # Combine plots into a single figure.
+        fig = bokeh.layouts.gridplot(
+            [fig1, fig2],
+            ncols=1,
+            toolbar_location="above",
+            merge_tools=True,
+            sizing_mode=sizing_mode,
```

Comment:
```suggestion
            sizing_mode=sizing_mode,
            toolbar_options=dict(active_inspect=None),
```

Apply solution from #633 to avoid warnings.